### PR TITLE
chore: replace `eslint-plugin-prettier` with `prettier-linter-helpers`

### LINKED ISSUE #264: Use prettier-linter-helpers
eslint-plugin-prettier v3 removed the generateDifferences and showInvisibles functions as they are now provided by the `prettier-linter-helpers` package. Depend upon that package instead of eslint-plugin-prettier.

[Here's the PR in stylelint-prettier](https://github.com/prettier/stylelint-prettier/pull/7] that changed from depending on eslint-plugin-prettier to prettier-linter-helpers to demonstrate the changes required.

### PR DESCRIPTION
**Pull request by JounQin**
Fixes #264

### CODE REVIEW

**Comment by JounQin** [comment]
@ikatyang Would you like help to review this PR?

File: package.json [hunk 1]
```diff
@@ -27,14 +27,14 @@
...
-    "eslint-plugin-prettier": "^2.2.0",
     "lines-and-columns": "^1.1.6",
+    "prettier-linter-helpers": "^1.0.0",
     "tslib": "^1.7.1"
   },
   "devDependencies": {
-    "@types/eslint-plugin-prettier": "2.2.0",
     "@types/node": "4.9.3",
     "@types/prettier": "2.0.0",
+    "@types/prettier-linter-helpers": "npm:@types/eslint-plugin-prettier",
```
[the remark this answers is not in the document]
Comment (JounQin) [reply package.json:37]: OK, I've draft https://github.com/DefinitelyTyped/DefinitelyTyped/pull/43776 for it.

File: package.json [hunk 2]
```diff
@@ -27,14 +27,14 @@
...
-    "eslint-plugin-prettier": "^2.2.0",
     "lines-and-columns": "^1.1.6",
+    "prettier-linter-helpers": "^1.0.0",
     "tslib": "^1.7.1"
   },
   "devDependencies": {
-    "@types/eslint-plugin-prettier": "2.2.0",
     "@types/node": "4.9.3",
     "@types/prettier": "2.0.0",
+    "@types/prettier-linter-helpers": "^1.0.0",
```
Comment (ikatyang) [inline package.json:37]: Exact version is preferred for devDeps in this repo:

```diff
-   "@types/prettier-linter-helpers": "^1.0.0",
+   "@types/prettier-linter-helpers": "1.0.0",
```

**Comment by ikatyang** [comment]
Thanks!

**Comment by JounQin** [comment]
@ikatyang Any chance to publish it to npm? On tag `next` for example?

**Comment by ikatyang** [comment]
@JounQin Any reason you want to use it right now? As far as I can tell, there is actually no difference between `prettier-linter-helpers` and `eslint-plugin-prettier`.

**Comment by JounQin** [comment]
@ikatyang I just think it's not reasonable to have two different versions of `eslint-plugin-prettier` installed which may cause unexpected behavior sometimes.

**Comment by ikatyang** [comment]
Make sense. I'll see if I have time to publish it tonight.

**Comment by ikatyang** [comment]
👉  https://github.com/prettier/tslint-plugin-prettier/blob/next/CHANGELOG.md#300-beta0-2020-04-21

**Pull request merged**